CHANGELOG.md

```diff
@@ -4,10 +4,7 @@
 
 ### Fixes
 
-### Features
-
-### Fixes
-
+- Installation id catches dir not exist([#1159](https://github.com/getsentry/sentry-dotnet/pull/1159))
```

Comment:
- Installation id catches dir not exist ([#1159](https://github.com/getsentry/sentry-dotnet/pull/1159))

We need to fix the missing space from Dange :)